### ISSUE TITLE
[FR] Add a hover background color on rows in queue

### ISSUE THREAD

**Issue by reloxx13** [sabnzbd/sabnzbd#1478]
Sometimes its hard to follow the rows from left file name to right actions, a hover color would help for a better usability in queue and history tables.

**Comment by thezoggy** [comment]
the row is already highlighted on mouse hover

**Comment by reloxx13** [comment]
Guess it depends on the theme, Glitter - Night does not have a hover.

**Comment by thezoggy** [comment]
you are correct, glitter-night does not. dunno what would be a good hover color that keeps things readable.. will have to play with some options

**Comment by Safihre** [comment]
Indeed a simple hover could help here. I thought I made that a while ago, but maybe it got lost.
Will see what I can do.

**Comment by reloxx13** [comment]
Git Beta2 and i like it, thank you. This helps alot!

**Comment by Safihre** [comment]
Funny nobody noticed it before in all these years, since the non-Night theme always had it 😅
